<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="peRcvMSG">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSendMSG">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>420</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leIpAddress"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePort"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>109</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="connectBTN">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disConnectBTN">
        <property name="text">
         <string>discocnnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendMsgBTN">
        <property name="text">
         <string>send  msg</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
